examples/pylint/C0102_blacklisted_name.py

```diff
@@ -0,0 +1,14 @@
+"""pylint: blacklisted name
+
+"""
+
+def pos(obj):
+    """
+    @type obj: int
+    @rtype: bool
+    """
+    foo = obj
+    if foo < 0:
```

Comment:
I notice a lot of your examples have the `if ...: return False else return True` form, which can be simplified to just `return ...`. Please change your examples.